ComposureWatch/src/Components/Card/CardForm.jsx

```diff
@@ -8,32 +8,34 @@ const CardForm = (props) => {
 
   const [selectedRank, setSelectedRank] = useState("Select Rank");
 
+  const [count, setCount] = useState(0);
+
   const nameChangeHandler = (event) => {
     setEnteredName(event.target.value);
-    console.log(selectedRank);
   };
 
   const selectCharacterHandler = (event) => {
     setSelectedCharacter(event.target.value);
-    console.log(selectedCharacter);
   };
 
   const selectRankHandler = (event) => {
     setSelectedRank(event.target.value);
-    console.log(selectedRank);
+  };
+
+  const handleClick = () => {
+    setCount(count + 1);
   };
 
   const submitHandler = (event) => {
     event.preventDefault();
 
     const charData = {
+      id: count,
```

Comment:
I think this works and is fine; however, this might be something you want to move up to your `CreateCard` component since it is the thing in control of the list of characters. If you ever had to re-create this form component by optionally rendering it or something, the useState() value would be reset and could create an issue with conflicting ids when you go to list characters.